rbi/annotations/lhm-shopify.rbi

```diff
@@ -0,0 +1,200 @@
+# typed: strong
+
+module Lhm
+  extend Lhm
+  extend Lhm::Throttler
+
+  sig do
+    params(
+      table_name: T.any(String, Symbol),
+      options: T::Hash[Symbol, T.untyped],
+      block: T.proc.params(arg0: Lhm::Migrator).void
+    ).returns(TrueClass)
+  end
+  def change_table(table_name, options = nil, &block); end
+
+  sig { params(run: T.nilable(T::Boolean), options: T.nilable(T::Hash[Symbol, T.untyped])).returns(T::Boolean) }
+  def cleanup(run = nil, options = nil); end
+
+  sig { returns(Lhm::Connection) }
+  def connection; end
+
+  sig { returns(Logger) }
+  def self.logger; end
+
+  sig { params(new_logger: Logger).returns(Logger) }
+  def self.logger=(new_logger); end
+
+  sig { params(adapter: Lhm::Connection).returns(Lhm::Connection) }
+  def setup(adapter); end
+end
+
+class Lhm::Table
+  sig { params(name: T.any(String, Symbol), pk: T.nilable(T.any(String, Symbol)), ddl: T.untyped).void }
+  def initialize(name, pk = nil, ddl = nil); end
+
+  sig { returns(T::Hash[T.untyped, T.untyped]) }
+  def columns; end
+
+  sig { returns(T.untyped) }
+  def ddl; end
+
+  sig { returns(String) }
+  def destination_name; end
+
+  sig { returns(T::Hash[T.untyped, T.untyped]) }
+  def indices; end
+
+  sig { returns(T.any(String, Symbol)) }
+  def name; end
+
+  sig { returns(T.any(String, Symbol)) }
+  def pk; end
+
+  sig { params(table_name: T.any(Symbol, String), connection: Lhm::Connection).returns(Lhm::Table) }
+  def self.parse(table_name, connection); end
+end
+
+module Lhm::Command
+  sig { params(block: T.nilable(T.proc.params(arg0: Lhm::Command).void)).void }
+  def run(&block); end
+end
+
+class Lhm::Printer::Output
+  sig { params(message: String).void }
+  def write(message); end
+end
+
+class Lhm::Printer::Base
+  sig { void }
+  def initialize; end
+end
+
+class Lhm::Printer::Percentage < Lhm::Printer::Base
+  sig { void }
+  def initialize; end
+
+  sig { void }
+  def end; end
+
+  sig { params(lowest: Numeric, highest: T.nilable(Numeric)).void }
+  def notify(lowest, highest); end
+end
+
+class Lhm::Printer::Dot < Lhm::Printer::Base
+  sig { void }
+  def end; end
+
+  sig { params(_arg0: T.untyped).void }
+  def notify(*_arg0); end
+end
```

Comment:
I've updated this method signature per [this thread](https://shopify.slack.com/archives/CBXRGT3SB/p1657915200780799).